hubspot-snowflake-aws-ml-insights/programmable-automation/eventCompletionWF-getEventCompletionData.js

```diff
@@ -0,0 +1,62 @@
+/*
+Workflow Custom Coded Action 1: Get HubSpot Event Completion Data and Transform it to send to AWS Personalize as Item Interactions
+Notes to make this function run:
+This is the first function in a HubSpot Workflow to get custom recommendations from AWS Personalize.
+
+Makes an API call to HubSpot Event Analytics API to get recent Custom Event Completions for the contact 
+
+Inputs:
+Set variables:
+1. Set HubSpot Access Token with proper scopes used by the HubSpot Workflow Custom Coded Action Secrete and used as environment variable
```

Comment:
Typo: `Secrete`

Also we may want this to be as straightforward as
> set "process.env.Token" in the workflow custom coded action secrets to be equal to your HubSpot private app token

Reply:
done